packages/content/src/Application/ContentResolver/Value/ResolvableResource.php

```diff
@@ -29,6 +29,7 @@ public function __construct(
         private int $priority,
         ?\Closure $resourceCallback = null,
         private ?array $metadata = null,
+        private ?string $resourceKey = null,
```

Comment:
@alexander-schranz I've added the resourceKey here, everything else would have been way too complicated to keep track of the map between a Reference and the ResolvableResource